desktop/libs/hadoop/src/hadoop/fs/webhdfs.py

```diff
@@ -896,6 +931,27 @@ def _get_redirect_url(self, webhdfs_ex):
       LOG.exception("Failed to read redirect from response: %s (%s)" % (webhdfs_ex, ex))
       raise webhdfs_ex
 
+
+  def _set_params_with_delegation_token(self, params):
```

Comment:
Is it possible to merge `_set_params_with_delegation_token` into `_getparams()`? That way we can keep the change locally in one place.

Reply:
Thank you for your comment. I merged the code. 🙇‍♀️